bugsnag/src/main/java/com/bugsnag/Bugsnag.java

```diff
@@ -556,8 +553,8 @@ public boolean shouldAutoCaptureSessions() {
      */
     @Deprecated
     public void setSessionEndpoint(String endpoint) {
-        if (config.sessionDelivery instanceof HttpDelivery) {
-            ((HttpDelivery) config.sessionDelivery).setEndpoint(endpoint);
+        if (config.getSessionDelivery() instanceof HttpDelivery) {
+            ((HttpDelivery) config.getSessionDelivery()).setEndpoint(endpoint);
```

Comment:
nit: it's probably a good idea to capture these to avoid calling twice

```suggestion
        Delivery delivery = config.getSessionDelivery()
        if (delivery instanceof HttpDelivery) {
            ((HttpDelivery) delivery).setEndpoint(endpoint);
```